app/tests/setup.js

```diff
@@ -0,0 +1,22 @@
+require('babel-register') ({
+  presets: ['es2015', 'react', 'stage-0']
```

Comment:
Hmm I thunk about it a bit further. 

Currently `webpack/rules/javascript`  is a function that accepts `production` or `browser` flags, which also returns `{}` that contains the rules required by webpack.

```js

var javascript = require('../../webpack/rules/javascript');
var babelOptions = javascript({ browser: true, production: true }).options;

require('babel-register') (babelOptions);
```

I initially intended to just use `webpack/rules` as an entry point (or interface) and did not see the need for javascript to be exposed outside. As a result, the javascript rules we are using is very coupled to webpack/rules. 

I'm not sure if accessing this `{}.options` is a good idea, because we are tightly coupling our  mocha setup to a webpack-specific rule, i.e. if webpack rules decide to accept something other than `option`, this setup would break. 

What we want is to actually use the common babel config that will be used in `webpack/rules/javascript` (which will transform the babel config into what webpack loves) as well as `setup.js`.

A logical place for a babel config file which exports presets and plugins should be `config/babel.js`.  However, I think this will only make the configuration wayyy more confusing. :(

What do you think? #overengineering?